ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -53,15 +53,15 @@ def shape(self):
     def dtype(self):
         return self.ivy_array.dtype
 
-    @property
-    def ndim(self):
-        return len(self.shape)
-
```

Comment:
unnecessary change

Reply:
Same thing here. I'm not sure how I changed the ndim() property. applying these changes right away :)